flw/pom.xml

```diff
@@ -121,6 +121,7 @@
                             org.springframework.transaction,
                             org.apache.commons.fileupload,
                             org.springframework.web.multipart.commons,
+                            org.motechproject.nms.region.repository,

```

Comment:
Why was the dependency on region.repository added.  It looks like it already existed and you just added a new dependency on service.  Can this be removed?